rundeckapp/grails-spa/packages/ui-trellis/src/app/components/readme-motd/EditProjectFile.spec.ts

```diff
@@ -135,31 +124,17 @@ describe("EditProjectFile", () => {
       "file.warning.not.displayed.nonadmin.message",
     );
   });
-
-  // This test checks if the save button is not displayed for non-admin users
   it("does not allow non-admin user to save the file", async () => {
     await mountEditProjectFile({ authAdmin: false });
     expect(wrapper.find('[data-test-id="save"]').exists()).toBe(false);
   });
-
-  // This test checks if the editor is not displayed for non-admin users
   it("does not allow non-admin user to edit the file", async () => {
     await mountEditProjectFile({ authAdmin: false });
     expect(wrapper.find('[data-test-id="editor"]').exists()).toBe(false);
   });
 
-  // This test checks if the Ace Editor is rendered
-  it("renders the Ace Editor", () => {
-    const aceEditor = wrapper.find('[data-test-id="ace-editor"]');
-    expect(aceEditor.exists()).toBe(true);
-  });
+  //TODO: working on this one
```

Comment:
@smartinellibenedetti Can you please look at it? This one not working. Thanks

Reply:
When you trigger actions that require the UI to re-render, remember to do a:

```
await wrapper.vm.$nextTick();
```

before trying to assert the values